### PR TITLE
An android Application with multiple sensors integrated

### PR DESCRIPTION
**Pull request by vintummala**
Signed-off-by: Vineela Tummalapalli <vineela.tummalapalli@intel.com>

### CODE REVIEW

**Comment by bjbeare** [comment]
I think that you should consider all of the output elements (gpio outputs, lcd display) as well as the framework display to be "user interface".

The run thread should read in the new temperature and button values from the UPM sensors and then call updateUI(). updateUI() would then update not only the framework graphics but also the LEDs and LCDs.

Bets to uncomment all of the commented out code so that this can be debugged and tested in terms of the I2C problem. We won't be merging it until we have that fixed.

**Comment by vintummala** [comment]
updateUI() has a runOnUiThread() which Runs the specified action on the UI thread. If the current thread is the UI thread, then the action is executed immediately. If the current thread is not the UI thread, the action is posted to the event queue of the UI thread. This thread is generally used to update the graphics on view on the UI thread. If i had to update the LEDs and LCDs, it can be part of UpdateUI() method but not part of runOnUiThread().
please let me know if is this still needed to be done?

**Comment by bjbeare** [comment]
Please rebase, verify that the formatting is correct and re-test.

**Comment by vintummala** [comment]
I updated this code as well. Bruce, can you please review the code.

File: multisensor/src/main/java/com/example/upm/androidthings/driversamples/MultiSensorActivity.java [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+                Blueled.delete();
+                touch.delete();
+                Greenled.off();
+                Greenled.delete();
+                MultiSensorActivity.this.finish();
+            }
+        }
+    };
+
+    @Override
```
Comment (bjbeare) [inline multisensor/src/main/java/com/example/upm/androidthings/driversamples/MultiSensorActivity.java]: don't we usually have onCreate before run task?

Comment (vintummala) [inline multisensor/src/main/java/com/example/upm/androidthings/driversamples/MultiSensorActivity.java]: When i did a reformat code from android studio, there was a re-arrange code checkbox checked in and so the run task is moved before oncreate(). This works fine as oncreate() is the first method to be executed. Tested the applications.
Also maintained consistency with all the other applications as well. As discussed with you, this code is good to be merged :)

**Pull request merged**